http4s/client/src/main/scala/endpoints4s/http4s/client/ChunkedEntities.scala

```diff
@@ -0,0 +1,73 @@
+package endpoints4s.http4s.client
+
+import org.http4s.{Request => Http4sRequest, Response => Http4sResponse}
+import fs2.Chunk
+
+import fs2.Stream
+import cats.syntax.all._
+import endpoints4s.Valid
+import endpoints4s.Invalid
+
+trait ChunkedEntities extends endpoints4s.algebra.ChunkedEntities with EndpointsWithCustomErrors {
+
+  //#stream-type
+  type Chunks[A] = fs2.Stream[Effect, A]
```

Comment:
Just one thought: it means that the type of an `Endpoint[A, Chunks[B]]` expands to `A => Effect[Stream[Effect, B]]`, right? Is that OK?

Reply:
Should we change the meaning of the type `Endpoint[A, B]` to the following?

~~~ scala
type Endpoint[A, B] = A => Effect[Resource[Effect, B]]
~~~

So that downstream users of the client could control the usage of the resource.

Reply:
Addressed by changing Endpoints to a trait that contains a convenience function that consumes the resource directly and one that exposes it